gpu/test/solver/cgs_kernels.cpp

```diff
@@ -73,7 +81,7 @@ class Cgs : public ::testing::Test {
         return gko::test::generate_random_matrix<Mtx>(
             ref, num_rows, num_cols,
             std::uniform_int_distribution<>(num_cols, num_cols),
-            std::normal_distribution<>(-1.0, 1.0), rand_engine);
+            std::normal_distribution<>(0.0, 1.0), rand_engine);
```

Comment:
Just to make sure (I have no idea), is this correct? The matrix should be positive?

Reply:
I think both should be fine.

Reply:
In this case maybe it's better to take the more general version and allow negative numbers.

Reply:
done.